tutorials/kube-mqtt/index.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Running a Cloud MQTT bridge in Kubernetes
+description: Bridge MQTT messages between a cluster and Cloud IoT Core with Mosquitto
+author: ptone
+tags: Kubernetes, iot, Internet of Things, cloud iot core
+date_published: 2019-07-01
+---
+
+Preston Holmes | Solution Architect | Google
+
+<!-- diagram sources: https://docs.google.com/presentation/d/1p2srfYUkqnXIKR4zGj099PIKOzz1RRi28eOcgIch1tc/edit#slide=id.p -->
+
+## Introduction
+
+This tutorial demonstrates how to deploy [Mosquitto](https://mosquitto.org/) MQTT broker in [Kubernetes](https://kubernetes.io/) using the broker's bridge feature to map a specific [MQTT](http://mqtt.org/) topic namespace to [Google Cloud IoT Core](https://cloud.google.com/iot-core/).
+
+
+## Setup
+
+### Clone the tutorial Repo
+#### TODO update to prod
+
+	git clone https://github.com/ptone/community.git
+	cd community
+	git checkout kube-mqtt
+	cd tutorials/kube-mqtt
+
+### Prerequisite: Kubernetes Cluster
+
+
+
+It is assumed you have running Kubernetes cluster. This can be via [Minikube](https://kubernetes.io/docs/tutorials/hello-minikube/), [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/), etc. For the tutorial, you can also use Kubernetes Engine clusters, however the context of this solution is based on non-cloud clusters in Edge environments.
+
+You will also need a recent `kubectl` command (>=1.14) with [kustomize integration](https://kubernetes.io/blog/2019/03/25/kubernetes-1-14-release-announcement/). If not, install it from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/). Ensure that Kubectl is configured to talk to your cluster.
```

Comment:
replace "If not install..." 

with:
"If you have an older version of `kubectl`, install a newer one from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/)."